packages/terra-badge/tests/specs/badge.js

```diff
@@ -0,0 +1,39 @@
+/* global browser, describe, it, expect, before, Terra */
+const viewports = Terra.viewports('tiny', 'medium');
+
+const shouldBeAccessible = () => {
```

Comment:
this is probably a candidate to move to toolkit... i just learned about these shared assertions here https://github.com/mochajs/mocha/wiki/Shared-Behaviours

Reply:
My vote is to also move these to toolkit.

Reply:
Do we want to set this on the Terra helper object?

Reply:
I don't have a strong preference but I would lean towards a Terra helper.

Reply:
Related PR: https://github.com/cerner/terra-toolkit/pull/29/files